[2024-02-12T23:00:17.412+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.create_snowflake_database scheduled__2024-02-12T22:30:00+00:00 [queued]>
[2024-02-12T23:00:17.430+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.create_snowflake_database scheduled__2024-02-12T22:30:00+00:00 [queued]>
[2024-02-12T23:00:17.431+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-02-12T23:00:17.508+0000] {taskinstance.py:2191} INFO - Executing <Task(SnowflakeOperator): create_snowflake_database> on 2024-02-12 22:30:00+00:00
[2024-02-12T23:00:17.522+0000] {standard_task_runner.py:60} INFO - Started process 326 to run task
[2024-02-12T23:00:17.532+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'create_snowflake_database', 'scheduled__2024-02-12T22:30:00+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/Weather_dag.py', '--cfg-path', '/tmp/tmpa0_kcsbe']
[2024-02-12T23:00:17.535+0000] {standard_task_runner.py:88} INFO - Job 143: Subtask create_snowflake_database
[2024-02-12T23:00:17.654+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.create_snowflake_database scheduled__2024-02-12T22:30:00+00:00 [running]> on host 3bd3b2605e24
[2024-02-12T23:00:17.828+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stellar' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='create_snowflake_database' AIRFLOW_CTX_EXECUTION_DATE='2024-02-12T22:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-12T22:30:00+00:00'
[2024-02-12T23:00:17.832+0000] {sql.py:276} INFO - Executing: 
                    CREATE DATABASE IF NOT EXISTS Project ;
            
[2024-02-12T23:00:17.854+0000] {base.py:83} INFO - Using connection ID 'conn_id_snowflake' for task execution.
[2024-02-12T23:00:17.871+0000] {base.py:83} INFO - Using connection ID 'conn_id_snowflake' for task execution.
[2024-02-12T23:00:17.873+0000] {connection.py:370} INFO - Snowflake Connector for Python Version: 3.6.0, Python Version: 3.8.18, Platform: Linux-5.15.133.1-microsoft-standard-WSL2-x86_64-with-glibc2.34
[2024-02-12T23:00:17.875+0000] {connection.py:1171} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-02-12T23:00:18.438+0000] {cursor.py:1028} INFO - query: [ALTER SESSION SET autocommit=False]
[2024-02-12T23:00:18.587+0000] {cursor.py:1041} INFO - query execution done
[2024-02-12T23:00:18.590+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-02-12T23:00:18.592+0000] {sql.py:450} INFO - Running statement: CREATE DATABASE IF NOT EXISTS Project ;, parameters: None
[2024-02-12T23:00:18.594+0000] {cursor.py:1028} INFO - query: [CREATE DATABASE IF NOT EXISTS Project ;]
[2024-02-12T23:00:18.679+0000] {cursor.py:1041} INFO - query execution done
[2024-02-12T23:00:18.680+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-02-12T23:00:18.681+0000] {sql.py:459} INFO - Rows affected: 1
[2024-02-12T23:00:18.682+0000] {snowflake.py:400} INFO - Rows affected: 1
[2024-02-12T23:00:18.682+0000] {snowflake.py:401} INFO - Snowflake query id: 01b24f84-3200-fe68-0007-a44e00010782
[2024-02-12T23:00:18.683+0000] {cursor.py:1028} INFO - query: [COMMIT]
[2024-02-12T23:00:18.761+0000] {cursor.py:1041} INFO - query execution done
[2024-02-12T23:00:18.762+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-02-12T23:00:18.764+0000] {connection.py:718} INFO - closed
[2024-02-12T23:00:18.825+0000] {connection.py:724} INFO - No async queries seem to be running, deleting session
[2024-02-12T23:00:18.922+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=create_snowflake_database, execution_date=20240212T223000, start_date=20240212T230017, end_date=20240212T230018
[2024-02-12T23:00:18.992+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-12T23:00:19.037+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
